docs/spec/sources.md

```diff
@@ -362,5 +370,10 @@ _Structured Content Mode_ of the HTTP Protocol Binding for CloudEvents. Sources
 SHOULD send events to its
 [Destination](https://pkg.go.dev/github.com/knative/pkg/apis/duck/v1?tab=doc#Destination).
 
+A source that generates or forwards CloudEvents to its destination MUST produce
```

Comment:
`SHOULD` more than `MUST`? Do we expect all sources, including `ContainerSource`, to fullfill this?

Reply:
As discussed during the WG, this is now a SHOULD with a sentence describing cases when this is not possible/desirable